apps/nowcasting-app/pages/index.tsx

```diff
@@ -237,26 +253,24 @@ export default function Home({ dashboardModeServer }: { dashboardModeServer: str
     }
   );
   // Load historic all GSP forecast data (once, at page load)
+  const forecastFrom = getEarliestForecastTimestamp();
   const {
     data: allGspForecastHistoricData,
     isLoading: allGspForecastHistoricLoading,
     isValidating: allGspForecastHistoricValidating,
     error: allGspForecastHistoricError
   } = useLoadDataFromApi<components["schemas"]["OneDatetimeManyForecastValues"][]>(
-    `${API_PREFIX}/solar/GB/gsp/forecast/all/?historic=true&compact=true&end_datetime_utc=${encodeURIComponent(
-      `${forecastLastFetch30MinISO.slice(0, 19)}+00:00`
-    )}`,
+    `${API_PREFIX}/solar/GB/gsp/forecast/all/?historic=true&compact=true&start_datetime_utc=${encodeURIComponent(
+      `${forecastFrom.slice(0, 19)}+00:00`
+    )}&end_datetime_utc=${encodeURIComponent(`${forecastLastFetch30MinISO.slice(0, 19)}+00:00`)}`,
     {
```

Comment:
could you add a comment that the `+00:00` is for the timezone? I miss read this first as `HH:MM`